near/src/main.rs

```diff
@@ -78,6 +78,7 @@ fn main() {
             .arg(Arg::with_name("network-addr").long("network-addr").help("Customize network listening address (useful for running multiple nodes on the same machine)").takes_value(true))
             .arg(Arg::with_name("rpc-addr").long("rpc-addr").help("Customize RPC listening address (useful for running multiple nodes on the same machine)").takes_value(true))
             .arg(Arg::with_name("telemetry-url").long("telemetry-url").help("Customize telemetry url").takes_value(true))
+            .arg(Arg::with_name("archive").long("archive").help("Keep old blocks in the storage (default false)").takes_value(true))
```

Comment:
Why is `takes_value=true`, if it's a boolean?
Look at `fast` parameter above for the `init` on line 66.

Separately, no need to provide the default, it is common sense that if there's a flag, than omitting it is the reverse of its behavior (whomever added `produce_empty_blocks` above for some reason also chose to make it a string parameter, but booleans should be `takes_value=false`, and not specify default value in the docs).